platform/docs/docs/platform/services/config-point-service.md

```diff
@@ -89,6 +94,44 @@ data, defaulting to the base behaviour of the application, allowing it to
 be exposed later to enhancements in a declarative fashion.  The remaining
 sections below simply expand on the idea with some more advanced concepts.
 
+## Why isn't Config-Point a general service?
```

Comment:
This addresses the comments about config-point being provided as the same type of service as the other services.  The argument is basically that it is too late to be provided at that point, but please read the description and see if it makes sense.